.github/workflows/verify-node.yml

```diff
@@ -12,9 +12,9 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - '14'
           - '16'
           - '18'
+          - '19'
```

Comment:
There seems to also be a `20.0.0-v8-canary` that we could try, but this seems like an easy enough things to bump forward when the ecosystem catches up over the next couple of months.

Reply:
i did try `'20'` but it complained there's no such version, not sure what format you use to get it to see the canary

Reply:
PRs/docs said it was that whole string `20.0.0-v8-canary`: https://github.com/actions/setup-node/blob/7c29869aec4da703a571b27bcd84d4f15af0b56e/docs/advanced-usage.md#install-v8-canary-build-for-major-node-version. Like I said, we can catch up again in a couple of weeks, either way.

Reply:
Ah I get you. IMO it makes sense to wait until it becomes just 20 (non canary). Happy to update it when that happens

Reply:
Sounds good!